### PR TITLE
feat: Source ClickBench queries from ClickBench

### PR DESCRIPTION
**Pull request by peasee**
# 🗣 Description

* Creates a script to source the ClickBench queries from the ClickBench repo, because the queries we have are not the same
* Adds overrides for SQLite to remove clickbench query 29, as SQLite does not support it.

## 🔨 Related Issues

* Part of #3953

### CODE REVIEW

File: crates/runtime/benches/queries/README.md [hunk 1]
```diff
@@ -427,3 +427,16 @@ select  cast(amc as FLOAT)/cast(pmc as FLOAT) am_pm_ratio
...
+## ClickBench
+
+### Attribution Notice
+
+The [ClickBench benchmark queries](./clickbench) are used under the [Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License (CC BY-NC-SA 4.0)](https://creativecommons.org/licenses/by-nc-sa/4.0/).
+
+- **Original Author(s):** [ClickBench Authors](https://github.com/ClickHouse/ClickBench)
+- **Source Repository:** [https://github.com/ClickBench/ClickBench](https://github.com/ClickHouse/ClickBench)
+- **License:** [CC BY-NC-SA 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/)
+- **Modifications:** This version use a subset of database sources, and makes the following modifications:
```
Comment (peasee) [inline crates/runtime/benches/queries/README.md]:
```suggestion
- **Modifications:** This version uses a subset of database sources, and makes the following modifications:
```

**Comment by peasee** [comment]
Need to update the script to make all the query column names wrapped in quotes `"`

**Review by phillipleblanc — APPROVED** [review]

**Comment by Sevenannn** [comment]
Did you test if the changed queries would work for DuckDB & Arrow? I added modifications to the original Clickbench query so that it could properly work with parquet data and Postgres syntax requirements, (see changes here: https://github.com/spiceai/spiceai/pull/3389), since Spice SQL is essentially a Postgres dialect SQL.

**Comment by peasee** [comment]
> Did you test if the changed queries would work for DuckDB & Arrow? I added modifications to the original Clickbench query so that it could properly work with parquet data and Postgres syntax requirements, (see changes here: #3389), since Spice SQL is essentially a Postgres dialect SQL.

Ideally, we can source the DuckDB queries from the ClickHouse benchmark and use those for DuckDB. However, those queries use mixed case column names that are unquoted which results in query errors because of ident normalization applied from our datafusion engine (and we cannot disable ident normalization).

**Pull request closed without merging**